cluster/local/local.sh

```diff
@@ -7,6 +7,36 @@ scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # shellcheck disable=SC1090
 projectdir="${scriptdir}/../.."
 
+# set cache directory
+cachedir="${projectdir}/.cache"
+
+# get os
+os=$(uname -s | tr '[:upper:]' '[:lower:]')
+if [ "$os" != "darwin" ] && [ "$os" != "linux"  ];then
+    echo ERROR: only supported on linux and darwin host currently
+    exit 1
+fi
+
+# get arch
+arch=$(uname -m)
+
+if [ "$arch" = "x86_64" ];then
+    arch="amd64"
+fi
+
+if [ "$arch" != "amd64" ];then
+    echo ERROR: only supported on amd64 host currently
+    exit 1
+fi
+
+hostplatform="${os}_${arch}"
+
+# set helm version
+helmver="v2.10.0"
+
+# set helm path
+helm="${cachedir}/tools/${hostplatform}/helm-${helmver}"
```

Comment:
I think it would be more robust for these variables to be exported out of the `make build.vars` (see https://github.com/upbound/build/blob/e66bc9d7d58cc0a4a14292a62f20d2163f9dcf3b/makelib/common.mk#L252) and the usage of below on line 41.

We could modify the build.vars target to export HELM_TOOL and this script here would just use it.

Reply:
Sounds good! Thanks for following up, will try to have this done ASAP